### ISSUE TITLE
Infrequent CTD on fast travel v3.1.0.0

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#211]
```
Unhandled native exception occurred at 0x7FFF92C98753 (SmartHarvestSE.dll+98753) on thread 51420!

FrameworkName: NetScriptFramework
FrameworkVersion: 9
FrameworkArchitecture: x64
GameLibrary: SkyrimSE
GameLibraryVersion: 13
ApplicationName: SkyrimSE.exe
ApplicationVersion: 1.5.97.0
VersionInfo: Successfully loaded
Time: 09 Sep 2020 01:59:09.211

Possible relevant objects (4)
{
[ 1] TESNPC(Name: `Nykiira`, FormId: 00000007, File: `Carry Weight - 500.esp <- Skyrim.esm`)
[ 1] PlayerCharacter(FormId: 00000014, BaseForm: TESNPC(Name: `Nykiira`, FormId: 00000007, File: `Carry Weight - 500.esp <- Skyrim.esm`))
[ 1] BGSLocation(Name: `Honningbrew Meadery`, FormId: 000DAB88, File: `Skyrim.esm`)
[ 34] TESObjectCELL(Name: DLC1VampireCastleExterior03, FormId: 000018FE, File: `Increased Enemy Spawns - 2x.esp`)
}

Probable callstack
{
[0] 0x7FFF92C98753 (SmartHarvestSE.dll+98753)
[1] 0x7FFFBB04321F (MSVCP140.dll+1321F)
[2] 0x7FFF92C85284 (SmartHarvestSE.dll+85284)
[3] 0x7FFFD0DEBB47 (ntdll.dll+BB47)
[4] 0x7FFF92C97894 (SmartHarvestSE.dll+97894)
[5] 0x7FFFBB04379E (MSVCP140.dll+1379E)
[6] 0x7FFF92C6AD59 (SmartHarvestSE.dll+6AD59)
[7] 0x7FFF92C6A769 (SmartHarvestSE.dll+6A769)
[8] 0x7FFF92C6B8DE (SmartHarvestSE.dll+6B8DE)
[9] 0x7FFFCEDB0E82 (ucrtbase.dll+20E82)
[10] 0x7FFFCF717BD4 (KERNEL32.DLL+17BD4)
[11] 0x7FFFD0E4CE51 (ntdll.dll+6CE51)
}

Registers
{
AX: 0x3F131FAC9D1E013D (u64):[4545011275177787709] (f64):[7.29512314858781E-05]
BX: 0x211FBC0A870 (BGSLocation*) -> (Name: `Honningbrew Meadery`, FormId: 000DAB88, File: `Skyrim.esm`)
CX: 0x211FD45CDE0 (void*)
DX: 0x7FFF92C5F44C (SmartHarvestSE.dll+5F44C) (void*)
SI: 0xA09FAFF8E0 (void*)
DI: 0x211FD45CDC0 (void*)
BP: 0xA09FAFF7A9 (void*)
SP: 0xA09FAFF740 (void*)
IP: 0x7FFF92C98753 (SmartHarvestSE.dll+98753) (void*)
R8: 0x7FFF92C00000 (SmartHarvestSE.dll+0) (char*) "MZ?"
R9: 0x1 (u:[1]
R10: 0x8000 (u16):[32768]
R11: 0xA09FAFF570 (void*)
R12: 0x2117D25D050 (PlayerCharacter*) -> (FormId: 00000014, BaseForm: TESNPC(Name: `Nykiira`, FormId: 00000007, File: `Carry Weight - 500.esp <- Skyrim.esm`))
R13: 0x211FD45CDC0 (void*)
R14: 0x212A65CD420 (void*)
R15: 0x0 (NULL)
Flags: 0x10202
XMM0: (double)5.19619574789592E-315 / (float)0.34375
XMM1: (double)5.6365513197515E-315 / (float)512
XMM2: (double)0 / (float)0
XMM3: (double)0 / (float)0
XMM4: (double)0 / (float)0
XMM5: (double)0 / (float)0
XMM6: (double)1 / (float)0
XMM7: (double)5 / (float)0
XMM8: (double)0.1 / (float)-1.588187E-23
XMM9: (double)0 / (float)0
XMM10: (double)0 / (float)0
XMM11: (double)0 / (float)0
XMM12: (double)0 / (float)0
XMM13: (double)0 / (float)0
XMM14: (double)0 / (float)0
XMM15: (double)0 / (float)0
}

Stack
{
[SP+0] 0xA09FAFF8C0 (char**) "SHSE_Logger"
[SP+8] 0x7FFFBB04321F (MSVCP140.dll+1321F) (void*)
[SP+10] 0x1D686877A4F74BE (u64):[132441155484742846]
[SP+18] 0x0 (NULL)
[SP+20] 0x21200000000 (void*)
[SP+28] 0xA09FAFF8E0 (void*)
[SP+30] 0x2123A894100 (void*)
[SP+38] 0x7FFF92C85284 (SmartHarvestSE.dll+85284) (void*)
[SP+40] 0x0 (NULL)
[SP+48] 0xF (u:[15]
[SP+50] 0x0 (NULL)
[SP+58] 0x212A5780550 (void*)
[SP+60] 0xA09FAFF7B0 (char**) "Scan progressing"
[SP+68] 0x7FFFD0DEBB47 (ntdll.dll+BB47) (void*)
[SP+70] 0x7FFF92DAB660 (SmartHarvestSE.dll+1AB660) (char*) "Scan progressing"
[SP+78] 0x10 (u:[16]
[SP+80] 0x212A57805A0 (void*)
[SP+88] 0x7FFFCEDB1301 (ucrtbase.dll+21301) (void*)
[SP+90] 0x7E7639D7946B (u64):[139046241670251] (f32):[0.0004111858]
[SP+98] 0x212A65CD420 (void*)
[SP+A0] 0x0 (NULL)
[SP+A8] 0x212A65CD420 (void*)
[SP+B0] 0x212A65CD4A8 (void*)
[SP+B8] 0x1 (u:[1]
[SP+C0] 0xA09FAFF8D9 (void*)
[SP+C8] 0x7FFF92C97894 (SmartHarvestSE.dll+97894) (void*)
[SP+D0] 0x211FBC0A870 (BGSLocation*) -> (Name: `Honningbrew Meadery`, FormId: 000DAB88, File: `Skyrim.esm`)
[SP+D8] 0x2112EFCE080 (void*)
[SP+E0] 0x2112EFCE080 (void*)
[SP+E8] 0x1 (u:[1]
[SP+F0] 0x1 (u:[1]
[SP+F8] 0x0 (NULL)
[SP+100] 0x211FD3E52C0 (TESObjectCELL*) -> (Name: DLC1VampireCastleExterior03, FormId: 000018FE, File: `Increased Enemy Spawns - 2x.esp`)
[SP+108] 0x2112EFCE080 (void*)
[SP+110] 0x211FD45CDC0 (void*)
[SP+118] 0x7FFFBB04379E (MSVCP140.dll+1379E) (void*)
[SP+120] 0x2123A894118 (char*) "SHSE_Logger"
[SP+128] 0xB (u:[11]
[SP+130] 0x34A100000002 (u64):[57866094379010]
[SP+138] 0x38D4A8A4785166 (u64):[15996419474739558]
[SP+140] 0xC8DC (u16):[51420]
[SP+148] 0x0 (NULL)
[SP+150] 0x0 (NULL)
[SP+158] 0x212A65CD4A8 (void*)
[SP+160] 0x0 (NULL)
[SP+168] 0x0 (NULL)
[SP+170] 0x7FFF92DAB678 (SmartHarvestSE.dll+1AB67 (char*) "UI/controls were active, delay scan"
[SP+178] 0x23 (u:[35]
[SP+180] 0x2123A894118 (char*) "SHSE_Logger"
[SP+188] 0xB (u:[11]
[SP+190] 0x34A100000002 (u64):[57866094379010]
[SP+198] 0x38D4A8A510F4BE (u64):[15996419484742846]
[SP+1A0] 0xC8DC (u16):[51420]
[SP+1A8] 0x0 (NULL)
[SP+1B0] 0x0 (NULL)
[SP+1B8] 0x0 (NULL)
[SP+1C0] 0x7E7639D79B3B (u64):[139046241671995] (f32):[0.0004112365]
[SP+1C8] 0x0 (NULL)
[SP+1D0] 0x7FFF92DA49B0 (SmartHarvestSE.dll+1A49B0) (char*) "SmartHarvestSE.esp"
[SP+1D8] 0x0 (NULL)
[SP+1E0] 0x0 (NULL)
[SP+1E8] 0x7FFF92DA5B08 (SmartHarvestSE.dll+1A5B0 (char*) "????????Plugin not initialized - Game Data load executing"
[SP+1F0] 0x0 (NULL)
[SP+1F8] 0x7FFF92C6AD59 (SmartHarvestSE.dll+6AD59) (void*)
[SP+200] 0x0 (NULL)
[SP+208] 0x1 (u:[1]
[SP+210] 0x0 (NULL)
[SP+218] 0x212A60FEF80 (BGSRefAlias**)
[SP+220] 0x0 (NULL)
[SP+228] 0x0 (NULL)
[SP+230] 0x0 (NULL)
[SP+238] 0x0 (NULL)
[SP+240] 0x0 (NULL)
[SP+248] 0x0 (NULL)
[SP+250] 0x7FFF92DA49B0 (SmartHarvestSE.dll+1A49B0) (char*) "SmartHarvestSE.esp"
[SP+258] 0x12 (u:[18]
[SP+260] 0x6C61767265746E00 (u64):[7809653462537629184]
[SP+268] 0x73646E6F636553 (u64):[32480047799690579]
[SP+270] 0x0 (NULL)
[SP+278] 0xF (u:[15]
[SP+280] 0x0 (NULL)
[SP+288] 0x0 (NULL)
[SP+290] 0x0 (NULL)
[SP+298] 0x0 (NULL)
[SP+2A0] 0x0 (NULL)
[SP+2A8] 0x0 (NULL)
[SP+2B0] 0x0 (NULL)
[SP+2B8] 0x0 (NULL)
[SP+2C0] 0x0 (NULL)
[SP+2C8] 0x0 (NULL)
[SP+2D0] 0x0 (NULL)
[SP+2D8] 0x0 (NULL)
[SP+2E0] 0x212F542ADA0 (void*)
[SP+2E8] 0x7FFF92C6A769 (SmartHarvestSE.dll+6A769) (void*)
[SP+2F0] 0x3FF0000000000000 (u64):[4607182418800017408] (f64):[1]
[SP+2F8] 0x0 (NULL)
[SP+300] 0x892FCF507D47 (u64):[150838434626887]
[SP+308] 0x212A60FEF80 (BGSRefAlias**)
[SP+310] 0x2129ED27D60 (void*)
[SP+318] 0x7FFF92C6B8DE (SmartHarvestSE.dll+6B8DE) (void*)
[SP+320] 0x0 (NULL)
[SP+328] 0x0 (NULL)
[SP+330] 0x0 (NULL)
[SP+338] 0x0 (NULL)
[SP+340] 0x2134E9E7EB0 (void*)
[SP+348] 0x7FFFCEDB0E82 (ucrtbase.dll+20E82) (void*)
[SP+350] 0x0 (NULL)
[SP+358] 0x2134E9E7EB0 (void*)
[SP+360] 0x0 (NULL)
[SP+368] 0x0 (NULL)
[SP+370] 0x0 (NULL)
[SP+378] 0x7FFFCF717BD4 (KERNEL32.DLL+17BD4) (void*)
[SP+380] 0x0 (NULL)
[SP+388] 0x0 (NULL)
[SP+390] 0x0 (NULL)
[SP+398] 0x0 (NULL)
[SP+3A0] 0x0 (NULL)
[SP+3A8] 0x7FFFD0E4CE51 (ntdll.dll+6CE51) (void*)
[SP+3B0] 0x0 (NULL)
[SP+3B8] 0x0 (NULL)
[SP+3C0] 0x0 (NULL)
[SP+3C8] 0x0 (NULL)
[SP+3D0] 0x0 (NULL)
[SP+3D8] 0x0 (NULL)
[SP+3E0] 0xE03A609F00000000 (i64):[-2289411224548409344]
[SP+3E8] 0x0 (NULL)
[SP+3F0] 0x0 (NULL)
[SP+3F8] 0x7FFFCE42F400 (KERNELBASE.dll+FF400) (void*)
[SP+400] 0xA09FAFE840 (void*)
[SP+408] 0x5CF1293E0000FFFE (u64):[6697179467141414910]
[SP+410] 0x7FFF2E78949F (u64):[140733973042335]
[SP+418] 0xA09FAFE840 (void*)
[SP+420] 0x0 (NULL)
[SP+428] 0x0 (NULL)
[SP+430] 0x0 (NULL)
[SP+438] 0x0 (NULL)
[SP+440] 0x0 (NULL)
[SP+448] 0x0 (NULL)
[SP+450] 0x0 (NULL)
[SP+458] 0x0 (NULL)
[SP+460] 0x0 (NULL)
[SP+468] 0x0 (NULL)
[SP+470] 0x0 (NULL)
[SP+478] 0x0 (NULL)
[SP+480] 0x0 (NULL)
[SP+488] 0x0 (NULL)
[SP+490] 0x0 (NULL)
[SP+498] 0x0 (NULL)
[SP+4A0] 0x0 (NULL)
[SP+4A8] 0x0 (NULL)
[SP+4B0] 0x0 (NULL)
[SP+4B8] 0x0 (NULL)
[SP+4C0] 0x0 (NULL)
[SP+4C8] 0x0 (NULL)
[SP+4D0] 0x0 (NULL)
[SP+4D8] 0x0 (NULL)
[SP+4E0] 0x0 (NULL)
[SP+4E8] 0x0 (NULL)
[SP+4F0] 0x0 (NULL)
[SP+4F8] 0x0 (NULL)
[SP+500] 0x0 (NULL)
[SP+508] 0x0 (NULL)
[SP+510] 0x0 (NULL)
[SP+518] 0x0 (NULL)
[SP+520] 0x0 (NULL)
[SP+528] 0x0 (NULL)
[SP+530] 0x0 (NULL)
[SP+538] 0x0 (NULL)
[SP+540] 0x0 (NULL)
[SP+548] 0x0 (NULL)
[SP+550] 0x0 (NULL)
[SP+558] 0x0 (NULL)
[SP+560] 0x0 (NULL)
[SP+568] 0x0 (NULL)
[SP+570] 0x0 (NULL)
[SP+578] 0x0 (NULL)
[SP+580] 0x0 (NULL)
[SP+588] 0x0 (NULL)
[SP+590] 0x0 (NULL)
[SP+598] 0x0 (NULL)
[SP+5A0] 0x0 (NULL)
[SP+5A8] 0x0 (NULL)
[SP+5B0] 0x0 (NULL)
[SP+5B8] 0x0 (NULL)
[SP+5C0] 0x0 (NULL)
[SP+5C8] 0x0 (NULL)
[SP+5D0] 0x0 (NULL)
[SP+5D8] 0x0 (NULL)
[SP+5E0] 0x0 (NULL)
[SP+5E8] 0x0 (NULL)
[SP+5F0] 0x0 (NULL)
[SP+5F8] 0x0 (NULL)
[SP+600] 0x0 (NULL)
[SP+608] 0x0 (NULL)
[SP+610] 0x0 (NULL)
[SP+618] 0x0 (NULL)
[SP+620] 0x0 (NULL)
[SP+628] 0x0 (NULL)
[SP+630] 0x0 (NULL)
[SP+638] 0x0 (NULL)
[SP+640] 0x0 (NULL)
[SP+648] 0x0 (NULL)
[SP+650] 0x0 (NULL)
[SP+658] 0x0 (NULL)
[SP+660] 0x0 (NULL)
[SP+668] 0x0 (NULL)
[SP+670] 0x0 (NULL)
[SP+678] 0x0 (NULL)
[SP+680] 0x0 (NULL)
[SP+688] 0x0 (NULL)
[SP+690] 0x0 (NULL)
[SP+698] 0x0 (NULL)
[SP+6A0] 0x0 (NULL)
[SP+6A8] 0x0 (NULL)
[SP+6B0] 0x0 (NULL)
[SP+6B8] 0x0 (NULL)
[SP+6C0] 0x0 (NULL)
[SP+6C8] 0x0 (NULL)
[SP+6D0] 0x0 (NULL)
[SP+6D8] 0x0 (NULL)
[SP+6E0] 0x0 (NULL)
[SP+6E8] 0x0 (NULL)
[SP+6F0] 0x0 (NULL)
[SP+6F8] 0x0 (NULL)
[SP+700] 0x0 (NULL)
[SP+708] 0x0 (NULL)
[SP+710] 0x0 (NULL)
[SP+718] 0x0 (NULL)
[SP+720] 0x0 (NULL)
[SP+728] 0x0 (NULL)
[SP+730] 0x0 (NULL)
[SP+738] 0x0 (NULL)
[SP+740] 0x0 (NULL)
[SP+748] 0x0 (NULL)
[SP+750] 0x0 (NULL)
[SP+758] 0x0 (NULL)
[SP+760] 0x0 (NULL)
[SP+768] 0x0 (NULL)
[SP+770] 0x0 (NULL)
[SP+778] 0x0 (NULL)
[SP+780] 0x0 (NULL)
[SP+788] 0x0 (NULL)
[SP+790] 0x0 (NULL)
[SP+798] 0x0 (NULL)
[SP+7A0] 0x0 (NULL)
[SP+7A8] 0x0 (NULL)
[SP+7B0] 0x0 (NULL)
[SP+7B8] 0x0 (NULL)
[SP+7C0] 0x0 (NULL)
[SP+7C8] 0x0 (NULL)
[SP+7D0] 0x0 (NULL)
[SP+7D8] 0x0 (NULL)
[SP+7E0] 0x0 (NULL)
[SP+7E8] 0x0 (NULL)
[SP+7F0] 0x0 (NULL)
[SP+7F8] 0x0 (NULL)
[SP+800] 0x0 (NULL)
[SP+808] 0x0 (NULL)
[SP+810] 0x0 (NULL)
[SP+818] 0x0 (NULL)
[SP+820] 0x0 (NULL)
[SP+828] 0x0 (NULL)
[SP+830] 0x0 (NULL)
[SP+838] 0x0 (NULL)
[SP+840] 0x0 (NULL)
[SP+848] 0x0 (NULL)
[SP+850] 0x0 (NULL)
[SP+858] 0x0 (NULL)
[SP+860] 0x0 (NULL)
[SP+868] 0x0 (NULL)
[SP+870] 0x0 (NULL)
[SP+878] 0x0 (NULL)
[SP+880] 0x0 (NULL)
[SP+888] 0x0 (NULL)
[SP+890] 0x0 (NULL)
[SP+898] 0x0 (NULL)
[SP+8A0] 0x0 (NULL)
[SP+8A8] 0x0 (NULL)
[SP+8B0] 0x0 (NULL)
[SP+8B8] 0x0 (NULL)
}

Modules
{
SkyrimSE.exe: 0x7FF79BCD0000
ntdll.dll: 0x7FFFD0DE0000
KERNEL32.DLL: 0x7FFFCF700000
KERNELBASE.dll: 0x7FFFCE330000
apphelp.dll: 0x7FFFCBF20000
USER32.dll: 0x7FFFD0420000
win32u.dll: 0x7FFFCDEE0000
GDI32.dll: 0x7FFFCF8F0000
gdi32full.dll: 0x7FFFCE010000
msvcp_win.dll: 0x7FFFCE210000
ucrtbase.dll: 0x7FFFCED90000
ADVAPI32.dll: 0x7FFFCF510000
msvcrt.dll: 0x7FFFD05C0000
sechost.dll: 0x7FFFD0AF0000
RPCRT4.dll: 0x7FFFCF7C0000
SHELL32.dll: 0x7FFFCFAE0000
cfgmgr32.dll: 0x7FFFCDF10000
shcore.dll: 0x7FFFD0CF0000
combase.dll: 0x7FFFD0730000
bcryptPrimitives.dll: 0x7FFFCE2B0000
windows.storage.dll: 0x7FFFCE5E0000
profapi.dll: 0x7FFFCDD30000
powrprof.dll: 0x7FFFCDCA0000
UMPDC.dll: 0x7FFFCDC90000
shlwapi.dll: 0x7FFFCF4B0000
kernel.appcore.dll: 0x7FFFCDCF0000
cryptsp.dll: 0x7FFFCED70000
ole32.dll: 0x7FFFD0B90000
WS2_32.dll: 0x7FFFD02A0000
SETUPAPI.dll: 0x7FFFCEFD0000
bcrypt.dll: 0x7FFFCDEB0000
OLEAUT32.dll: 0x7FFFD01D0000
X3DAudio1_7.dll: 0x50410000
XINPUT1_3.dll: 0x400000
DINPUT8.dll: 0x7FFFC0440000
binkw64.dll: 0x180000000
WINHTTP.dll: 0x7FFFAC350000
COMDLG32.dll: 0x7FFFD0660000
IPHLPAPI.DLL: 0x7FFFCD200000
steam_api64.dll: 0x503A0000
MSVCP140.dll: 0x7FFFBB030000
WSOCK32.dll: 0x7FFFBE970000
d3dx9_42.dll: 0x7FFFCA5B0000
VCRUNTIME140.dll: 0x7FFFCA510000
d3d11.dll: 0x7FFFCB0A0000
HID.DLL: 0x7FFFCC890000
WINMM.dll: 0x7FFFCBEC0000
dxgi.dll: 0x7FFF820E0000
COMCTL32.dll: 0x7FFFC11C0000
tbbmalloc.dll: 0x7FFFC02B0000
UxTheme.dll: 0x7FFFCC280000
tbb.dll: 0x7FFFB8A60000
VCRUNTIME140_1.dll: 0x7FFFCA5E0000
WINMMBASE.dll: 0x7FFFCA940000
WINHTTP.DLL: 0x7FFFC1430000
WININET.dll: 0x7FFFBA7D0000
IMM32.DLL: 0x7FFFCF660000
skse64_steam_loader.dll: 0x7FFFC67C0000
VERSION.dll: 0x7FFFC76F0000
inputhost.dll: 0x7FFFAF850000
wintypes.dll: 0x7FFFC9720000
CoreMessaging.dll: 0x7FFFCBCB0000
PROPSYS.dll: 0x7FFFCA630000
CoreUIComponents.dll: 0x7FFFC93F0000
ntmarta.dll: 0x7FFFCCE40000
steamclient64.dll: 0x7FFF80DB0000
CRYPT32.dll: 0x7FFFCDD60000
MSASN1.dll: 0x7FFFCDD10000
imagehlp.dll: 0x7FFFCF5C0000
PSAPI.DLL: 0x7FFFCF8E0000
vstdlib_s64.dll: 0x7FFFAD980000
tier0_s64.dll: 0x7FFFAC1D0000
MSWSOCK.dll: 0x7FFFCD4F0000
Secur32.dll: 0x7FFFB8130000
SSPICLI.DLL: 0x7FFFCDB80000
CRYPTBASE.DLL: 0x7FFFCD6C0000
FPSFixPlugin.dll: 0x7FFFC0600000
NetScriptFramework.Runtime.dll: 0x7FFFBEA40000
dbghelp.dll: 0x7FFFB7DB0000
mscoree.dll: 0x7FFFBE760000
mscoreei.dll: 0x7FFFBE160000
clr.dll: 0x7FFFBCB30000
VCRUNTIME140_CLR0400.dll: 0x7FFFBC4C0000
ucrtbase_clr0400.dll: 0x7FFFBC4E0000
mscorlib.ni.dll: 0x7FFF853B0000
clrjit.dll: 0x7FFFB42B0000
wldp.dll: 0x7FFFCD750000
WINTRUST.dll: 0x7FFFCE1B0000
amsi.dll: 0x7FFFB10E0000
USERENV.dll: 0x7FFFCDBB0000
MpOav.dll: 0x7FFFB1060000
MPCLIENT.DLL: 0x7FFFBB5F0000
gpapi.dll: 0x7FFFCC8A0000
System.ni.dll: 0x7FFF84740000
System.Core.ni.dll: 0x7FFFAEDD0000
urlmon.dll: 0x7FFFBF480000
iertutil.dll: 0x7FFFBF1D0000
NetScriptFramework.SkyrimSE.dll: 0x7FFFAA780000
EngineFixes.dll: 0x7FFFAA980000
skse64_1_5_97.dll: 0x7FFFA3E20000
dbgcore.DLL: 0x7FFFB6970000
AddItemMenuSE.dll: 0x7FFFAD6D0000
AlternateConversationCamera.dll: 0x7FFFAD930000
D3DCOMPILER_47.dll: 0x7FFFCB300000
auto_backup_executable.dll: 0x7FFF93BD0000
cbp.dll: 0x7FFFAC120000
DisplayEnemyLevel.dll: 0x7FFFACBB0000
encounter_zones_unlocked.dll: 0x7FFFAD670000
Fuz Ro D'oh.dll: 0x7FFFABAD0000
IndividualShoutCooldownRemake.dll: 0x7FFFAB750000
JContainers64.dll: 0x7FFF80A70000
rsaenh.dll: 0x7FFFCD060000
mfgfix.dll: 0x7FFFAADA0000
PapyrusUtil.dll: 0x7FFF96F60000
po3_PapyrusExtender.dll: 0x7FFFAA5D0000
po3_SpellPerkItemDistributor.dll: 0x7FFFAD190000
SchlongsOfSkyrim.dll: 0x7FFFAA6F0000
skee64.dll: 0x7FFF94060000
SmartHarvestSE.dll: 0x7FFF92C00000
SpeedCastingRemake.dll: 0x7FFFA8050000
TKPlugin.dll: 0x7FFFBB000000
to_your_face.dll: 0x7FFFAB180000
YeOldePapyrus.dll: 0x7FFFAA590000
gameoverlayrenderer64.dll: 0x7FFF92310000
MSCTF.dll: 0x7FFFCEE90000
TextInputFramework.dll: 0x7FFFB5F20000
dxgi.dll: 0x7FFFCC980000
dxcore.dll: 0x7FFFCC8F0000
nvldumdx.dll: 0x7FFFC8A40000
nvwgf2umx.dll: 0x7FFF869B0000
nvspcap64.dll: 0x7FFF80070000
ondemandconnroutehelper.dll: 0x7FFFB2830000
NSI.dll: 0x7FFFCF440000
WINNSI.DLL: 0x7FFFC6790000
DPAPI.DLL: 0x7FFFCD0A0000
DNSAPI.dll: 0x7FFFCD240000
rasadhlp.dll: 0x7FFFC10B0000
fwpuclnt.dll: 0x7FFFC4EC0000
schannel.DLL: 0x7FFFCCF90000
mskeyprotect.dll: 0x7FFF991B0000
ncrypt.dll: 0x7FFFCD7C0000
NTASN1.dll: 0x7FFFCD780000
cryptnet.dll: 0x7FFFC5170000
ncryptsslp.dll: 0x7FFF9B3A0000
DEVOBJ.dll: 0x7FFFCDA90000
XAudio2_7.dll: 0x7FFFA18D0000
clbcatq.dll: 0x7FFFD0370000
MMDevApi.dll: 0x7FFFC5C40000
AUDIOSES.DLL: 0x7FFFC5DA0000
Windows.UI.dll: 0x7FFFAB920000
avrt.dll: 0x7FFFCA080000
DSOUND.DLL: 0x7FFFA91D0000
dwmapi.dll: 0x7FFFCC5F0000
igd10iumd64.dll: 0x7FFFC3A30000
igc64.dll: 0x7FFFC2420000
OneCoreCommonProxyStub.dll: 0x7FFFB81E0000
OneCoreUAPCommonProxyStub.dll: 0x7FFFC6F80000
dcomp.dll: 0x7FFFCB760000
}

Plugins (1)
{
bug_fixes
{
Name: "Bug Fixes SSE"
Version: 1
Author: "meh321"
Assembly: BugFixesSSE, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
}
}

Extra
{
Currently executing hooks (0)
{
}
}

Game plugins (142)
{
[00] Skyrim.esm
[01] Update.esm
[02] Dawnguard.esm
[03] HearthFires.esm
[04] Dragonborn.esm
[05] Unofficial Skyrim Special Edition Patch.esp
[06] Schlongs of Skyrim - Core.esm
[07] nwsCVRFollower.esp
[08] FlowerGirls SE.esm
[09] Schlongs of Skyrim.esp
[0A] SOS - VectorPlexus Regular Addon.esp
[0B] SOS - VectorPlexus Muscular Addon.esp
[0C] TKDodge.esp
[0D] FNIS.esp
[0E] AsharaParagonJewelry.esp
[0F] Increased Enemy Spawns - 2x.esp
[10] HF_Lakeview_AvantGarden_EX.esp
[11] FeMequins - Hearthfire.esp
[12] Quest Conflict Fixes.esp
[13] Diverse Males.esp
[14] UIExtensions.esp
[15] AddItemMenuSE.esp
[16] Simply Stronger Dragons.esp
[17] Males of Skyrim.esp
[18] PAN_NPCs_DB.esp
[19] PAN_NPCs_DG.esp
[1A] Draugr Upgrades and Improvements.esp
[1B] HF_Windstad_PHB_EX.esp
[1C] HF_Heljarchen_PHBXB1_EX.esp
[1D] FlowerGirls SE - Adventures.esp
[1E] Triumvirate - Mage Archetypes.esp
[1F] Whiterun_UndergroundLair.esp
[20] AmorousAdventuresPlus.esp
[21] WhiterunSafehold.esp
[22] svelkfollower.esp
[23] StrangeRunes.esp
[24] Brows.esp
[25] ScarletDawnArmor.esp
[26] Holiday Gift Hairdos.esp
[27] VioLens SE.esp
[28] WetFunction.esp
[29] DX Druid Armor.esp
[2A] Cumshot.esp
[2B] Ghosu - Project Flintlock.esp
[2C] Elemental Destruction Magic Redux.esp
[2D] Umbriel - The Last Sea Elf.esp
[2E] YeOlde - Crafting Bag.esp
[2F] FGCumEffect.esp
[30] Vidani's Bag of Holding.esp
[31] AutoConsumePotions.esp
[32] ExperienceMultipliers.esp
[33] DamageScalingWithLevel.esp
[34] SoulGemVendor.esp
[35] TES5SE NPCs Travel.esp
[36] tetro - Lilu.esp
[37] Cyrus.esp
[38] PAN_BlackBriar.esp
[39] TERAArmorsSSE.esp
[3A] PAN_NPCs.esp
[3B] Giantess.esp
[3C] FemaleWerewolfExtended.esp
[3D] CT_AskToShowRoom.esp
[3E] Bijin_AIO 2020.esp
[3F] FlowerGirls SE - Exotic Flowers.esp
[40] PAN_Serana.esp
[41] UD_Serana.esp
[42] BW Ria.esp
[43] BW Aela.esp
[44] Bijin_Frea-Iona.esp
[45] CrownHelmetsRedux_Replacer.esp
[46] MIDSM - Modified Improved Dragon Shouts Modified.esp
[47] PrvtI_HeavyArmory.esp
[48] Heritage.esp
[49] PAN_Valerica.esp
[4A] Perk Points at Skill Levels 50-75-100.esp
[4B] NosPiercedEarsAll.esp
[4C] PyP_Legendary.esp
[4D] ogSplendorDragons.esp
[4E] Vokrii - Minimalistic Perks of Skyrim.esp
[4F] Alternate Start - Live Another Life.esp
[50] Relationship Dialogue Overhaul.esp
[51] Serana Dialogue Edit.esp
[52] Alternate Start -- New Beginnings.esp
[53] nwsFollowerFramework.esp
[FE 000] FlowerGirls SE Threesome Fix.esl
[FE 001] Morningstar - Minimalistic Races of Skyrim.esl
[FE 002] SimpleChildren.esp
[FE 003] SkyUI_SE.esp
[FE 004] Mortal Enemies - Rival Remix - NoRunWalkChanges.esp
[FE 005] SmartHarvestSE.esp
[FE 006] Rayg_NoBYOHAttacks.esp
[FE 007] Enhanced Enemy AI.esp
[FE 008] FugitivePissOff.esp
[FE 009] Radiant Quest Marker SE.esp
[FE 00A] Radiant Quest Marker SE QCF Patch.esp
[FE 00B] More Feminine Females - All.esp
[FE 00C] DraugrShoutLess.esp
[FE 00D] Perk Point Awards.esp
[FE 00E] Increased Enemy Bosses - 2x.esp
[FE 00F] MortalEnemies_DraugrImprovements_Patch.esp
[FE 010] Arena.esp
[FE 011] NPC Magic and Constant Spells Rebalance.esp
[FE 012] Proper Aiming.esp
[FE 013] RaceMenuNails.esp
[FE 014] YMMP.esp
[FE 015] CumshotFG.esp
[FE 016] AdventurerFlintlockPack.esp
[FE 017] Quests Award Perk Points.esp
[FE 018] Carry Weight - 500.esp
[FE 019] Magicka_Cost_Level_Scaling_90.esp
[FE 01A] Nilheim.esp
[FE 01B] NoRadialBlur.esp
[FE 01C] TESL-LoadingScreens.esp
[FE 01D] dD-No Spinning Death Animation Merged.esp
[FE 01E] NoBSAIProjectileDodge.esp
[FE 01F] FreckleMania2.esp
[FE 020] RaceMenu.esp
[FE 021] AstridAfflictedUNP.esp
[FE 022] UUNP Vanilla Outfits.esp
[FE 023] aMidianBorn_Book of Silence.esp
[FE 024] XPMSE.esp
[FE 025] CommunityOverlays3.esp
[FE 026] CommunityOverlays2_31T50.esp
[FE 027] CommunityOverlays1_0T30.esp
[FE 028] Female Argonian Tail Patch.esp
[FE 029] icepenguinworldmapclassic.esp
[FE 02A] Better Skill and Quest Books Names SE.esp
[FE 02B] Better Face Lighting.esp
[FE 02C] PAN_Initiates.esp
[FE 02D] Easy lockpicking.esp
[FE 02E] Less Tedious Thieves Guild.esp
[FE 02F] sandboxcylinderheight.esp
[FE 030] RichMerchantsSkyrim_x5.esp
[FE 031] RealisticHuskySoundsSSE.esp
[FE 032] RealisticDogSoundsSSE.esp
[FE 033] FacegenForKids.esp
[FE 034] Persuasion Check Actual Value.esp
[FE 035] Realistic Haircolors.esp
[FE 036] Dibella Priestess Outfits.esp
[FE 037] Vanilla Follower AI Overhaul.esp
[FE 038] DawnguardDragonbornTiming.esp
[FE 039] Modern Brawl Bug Fix.esp```

**Comment by SteveTownsend** [comment]
Got this in the debugger using JSONTest updated to load the DLL. The crash is via
```
		std::string oldName(PlaceName(originalPlace));
```
I am reworking the refresh logic in this area to clean it up.

**Comment by SteveTownsend** [comment]
v3.1.0.0 adds some possible mitigation but I am not totally convinced it will fix this.

**Comment by SteveTownsend** [comment]
This is less frequent in v3.1.0.0 per Nexus user update

> Just had the same crash again. This time when fast traveling to a forsworn camp that had respawned, so I was in action immediately upon arrival. It's super infrequent now, but still happening. Honestly it's pretty easy to live with at this point. Before it was frequent enough to stress me out every time I fast traveled.

**Comment by SteveTownsend** [comment]
This appears to be caused by accessing CELL records after they have been unloaded by the game engine. Reworked Location Tracker to use From ID instead of pointers. Fix released in v3.2.0.1.
